aws/scylla_create_devices

```diff
@@ -18,16 +18,10 @@ import re
 import os
 import sys
 import time
-import logging
 import subprocess
 import urllib.request
 import urllib.error
 from pathlib import Path
-from lib.log import setup_logging
-
-
-setup_logging()
```

Comment:
why are we removing logging for this one ?

Reply:
@bentsi I am also curious -- why is logging being removed? What's the impact of this to existing systems?

Reply:
@bentsi, ping?

Reply:
`scylla_create_devices` is a simple script that can be run outside the scylla-machine-image setup, it shouldn't use the Scylla Machine Image (SMI) log. The log was created for SMI configurator and not for the scylla-image-setup.service (it uses the systemd facility)